Palo Alto Networks/paloalto-ngfw/ingest/parser.yml

```
@@ -794,6 +794,9 @@ stages:
           network.packets: "{{parsed_event.message.Packets or parsed_event.message.PacketsTotal}}"
         filter: "{{ (parsed_event.message.get('PacketsTotal') != None and (parsed_event.message.PacketsTotal is number or parsed_event.message.PacketsTotal.isdigit())) or (parsed_event.message.get('Packets') != None and (parsed_event.message.Packets is number or parsed_event.message.Packets.isdigit())) }}"
 
+      - set:
+          network.name: "{{parsed_event.message.SourceZone}}"
```

I suggest using [`observer.ingress.vlan.name`](https://www.elastic.co/guide/en/ecs/8.17/ecs-vlan.html) instead (as we can also extract DestinationZone later and `network.name` seems to be not suitable for this: What is the name of the network between SourceZone and DestinationZone?):
```suggestion
          observer.ingress.vlan.name: "{{parsed_event.message.SourceZone}}"
```